src/platforms/javascript/common/sourcemaps/index.mdx

```diff
@@ -98,6 +98,20 @@ From here, you can [upload your source maps](uploading/) to Sentry.
 
 </PlatformSection>
 
+<PlatformSection supported={["javascript.nextjs"]}>
+
+You need to generate an access token for our API. Within your organization's settings, navigate to Developer Settings, [create a new internal integration](/product/integrations/integration-platform/#internal-integrations), and provide a name appropriate to your organization. **Important:** Select **Releases -> Admin** for Permissions.
+
+<Note>
+
+The Releases -> Admin permission is also known as 'project:releases' in other API documentation.
+
+</Note>
+
+You may configure [sentry-cli](/product/cli/configuration/) through its documented mechanisms, or follow the [recommended instructions for webpack](/platforms/javascript/guides/nextjs/sourcemaps/tools/webpack/) for Next.js.
```

Comment:
I don't know for sure what's the best practice at the moment, but a few months back we were preferring relative links. @PeloWriter, could you please confirm?

```suggestion
You may configure [sentry-cli](/product/cli/configuration/) through its documented mechanisms, or follow the [recommended instructions for webpack](tools/webpack/) for Next.js.
```

Reply:
When you're linking to common content that you want to display for a specific language, you need to use an absolute link (for example, /platforms/javascript/enriching-events/breadcrumbs/ will link to the JS content for breadcrumbs). If you want to display for a specific language, but need the context of the language to drive the display, you use our platform redirect ((/platform-redirect/?next=/enriching-events/breadcrumbs/) will link to the content for breadcrumbs depending on the platform you're viewing (or direct you to the platform picker if you don't yet have a platform selected).

Here, we're using content that is common within the JS family (it lives in javascript/common/sourcemaps/tools/webpack/). As a result, you can use a relative link (as we do at the top of the file when listing the tools available) or an absolute link. Markus' link checker will find any breakage should content be moved in  future, so either choose is safe.

Reply:
I'll go with absolute links, I find them more straightforward and easier to read.